packaging/vagrant/Vagrantfile

```diff
@@ -191,6 +183,57 @@ EOF2
     ch-run --version
   EOF
 
+  # Install Buildah.
+  c.vm.provision "buildah", type: "shell", privileged: false,
+                 inline: <<-EOF
+    set -e
+    cd /usr/local/src
+
+    # CentOS/EPEL/IUI don't have the version of shadow-utils (newuidmap and
+    # newgidmap) we need for runc (used by Buildah), so install from source.
+    wget -nv https://github.com/shadow-maint/shadow/releases/download/4.6/shadow-4.6.tar.xz
+    tar xf shadow-4.6.tar.xz
+    (cd shadow-4.6 && ./configure && sudo make install)
+
+    sudo yum -y install bats \
+                        btrfs-progs-devel \
+                        bzip2 \
+                        device-mapper-devel \
+                        glib2-devel \
+                        golang \
+                        go-md2man \
+                        gpgme-devel \
+                        libassuan-devel \
+                        libseccomp-devel \
+                        make \
+                        ostree-devel \
+                        runc
+
+    mkdir buildah
```

Comment:
Why `/usr/local/src/buildah` instead of `/usr/local/go`?

Reply:
I'm not quite sure. Perhaps @rstyd can shed some light on this, I believe he wrote the buildah portion of the install.

FWIIW, we are building buildah from git source (to specify `v1.11.2`) and not `go build`.